facia/app/views/fragments/emailFrontBody.scala.html

```diff
@@ -52,10 +52,6 @@
         }
     }
 
-    @if(isLast) {
-        @row(Seq("fc__pad")) { }
```

Comment:
This gets applied to all, not just the first so we don't care about isLast anymore